string-manipulation-1/src/get-last-char.js

```diff
@@ -0,0 +1,6 @@
+/* exported getLastChar */
+
+function getLastChar(string) {
+  var lastChar = string.charAt(string.length - 1);
+  return lastChar;
```

Comment:
You can also accomplish this using bracket notation, rather than using `charAt`
```
var lastChar = string.length - 1;
  return string[lastChar];
```